### PR TITLE
Mailmodo destination doc

### PR DESCRIPTION
**Pull request by ishachopra0401**
## What do these changes do?

Created Mailmodo destination doc

### CODE REVIEW

File: docs/destinations/streaming-destinations/mailmodo.mdx [hunk 1]
```diff
@@ -0,0 +1,140 @@
+---
+title: "Mailmodo"
+description: Step-by-step guide on sending your event data from RudderStack to Mailmodo.
+aliases: ["Mailmodo destination"]
+---
+
+[Mailmodo](https://www.mailmodo.com/) is an interactive email marketing tool that provides several useful features such as, sending transactional campaigns, email automation and drip emails with customer journeys, WYSIWYG email editor and many other tool integrations.
+
+RudderStack supports Mailmodo as a destination where you can seamlessly send your event data.
```
Comment (rohithkaza) [inline docs/destinations/streaming-destinations/mailmodo.mdx:9]: We are supporting Mailmodo as a source too. Do we need this line here?

File: docs/destinations/streaming-destinations/mailmodo.mdx [hunk 2]
```diff
@@ -0,0 +1,140 @@
...
+
+<img src="../../assets/event-stream-destinations/mailmodo-connection-settings.png" alt="Mailmodo connection settings" />
+
+- **API Key**: Enter your Mailmodo API key.
+
+<div class="infoBlock">
+For more information on obtaining your Mailmodo API key, refer to the <Link to="#FAQ">FAQ</Link> section below.
+</div>
+
+- **List Name**: This field is **only applicable** for the `identify` call. Enter the name of the Mailmodo list where contacts should be added. If not provided, `rudderstack` is taken as the default list name.
```
Comment (rohithkaza) [inline docs/destinations/streaming-destinations/mailmodo.mdx:41]:
```suggestion
- **List Name**: This field is **only applicable** for the `identify` call. Enter the name of the Mailmodo list where contacts should be added. If not provided, `Rudderstack` is taken as the default list name. 
```

File: docs/destinations/streaming-destinations/mailmodo.mdx [hunk 3]
```diff
@@ -0,0 +1,154 @@
...
+| `traits.name`/`context.traits.name` | `values.data.name` | Optional |
+| `traits.gender`/`context.traits.gender` | `values.data.gender` | Optional |
+| `traits.age`/`context.traits.age` | `values.data.age` | Optional (Integer format) |
+| `traits.birthday`/`context.traits.birthday` | `values.data.birthday` | Optional (ISO format/UNIX timestamp) |
+| `traits.phone`/`context.traits.phone` | `values.data.phone` | Optional |
+| `traits.city`/`context.traits.city`/`traits.address.city`/`context.traits.address.city` | `values.data.city` | Optional |
+| `traits.address`/`context.traits.address` | `address1` + `address2` | Optional |
+| `traits.state`/`context.traits.state`/`traits.address.state`/`context.traits.address.state` | `values.data.state` | Optional |
+| `traits.country`/`context.traits.country`/`traits.address.country`/`context.traits.address.country` | `values.data.country` | Optional |
+| `traits.zipcode`/`context.traits.zipcode` | `values.data.postal_code` | Optional |
```
Comment (rohithkaza) [inline docs/destinations/streaming-destinations/mailmodo.mdx:100]: lets add `postalcode` and `postal_code`

File: docs/destinations/streaming-destinations/mailmodo.mdx [hunk 4]
```diff
@@ -0,0 +1,154 @@
...
+
+### Supported mappings
+
+The following table lists the mappings between the RudderStack and Mailmodo properties:
+
+| RudderStack property | Mailmodo property | Presence |
+| :--------------| :---------| :--------------|
+| `traits.email`/`context.traits.email`/`properties.email` | `values.email` | Required |
+| `traits.firstName`/`context.traits.firstName` | `values.data.first_name` | Optional |
+| `traits.lastName`/`context.traits.lastName` | `values.data.lastName` | Optional |
```
Comment (rohithkaza) [inline docs/destinations/streaming-destinations/mailmodo.mdx:90]:
```suggestion
| `traits.lastName`/`context.traits.lastName` | `values.data. last_name ` | Optional |
```

File: docs/destinations/streaming-destinations/mailmodo.mdx [hunk 5]
```diff
@@ -0,0 +1,154 @@
+---
+title: "Mailmodo"
+description: Step-by-step guide on sending your event data from RudderStack to Mailmodo.
+aliases: ["Mailmodo destination"]
+---
+
+[Mailmodo](https://www.mailmodo.com/) is an interactive email marketing tool that provides several useful features such as, sending transactional campaigns, email automation and drip emails with customer journeys, WYSIWYG email editor and many other tool integrations.
```
Comment (ItsSudip) [inline docs/destinations/streaming-destinations/mailmodo.mdx:7]:
```suggestion
[Mailmodo](https://www.mailmodo.com/) is an interactive email marketing tool that provides several useful features such as sending transactional campaigns, email automation and drip emails with customer journeys, WYSIWYG email editor, and many other tool integrations.
```

**Review by ItsSudip — CHANGES_REQUESTED** [review]

**Review by rohithkaza — APPROVED** [review]

File: docs/destinations/streaming-destinations/mailmodo.mdx [hunk 6]
```diff
@@ -0,0 +1,158 @@
+---
+title: "Mailmodo"
+description: Step-by-step guide on sending your event data from RudderStack to Mailmodo.
+aliases: ["Mailmodo destination"]
+---
+
+[Mailmodo](https://www.mailmodo.com/) is an interactive email marketing tool that provides several useful features such as sending transactional campaigns, email automation and drip emails with customer journeys, WYSIWYG email editor, and many other tool integrations.
```
Comment (ameypv-rudder) [inline docs/destinations/streaming-destinations/mailmodo.mdx:7]:
```suggestion
[Mailmodo](https://www.mailmodo.com/) is an interactive email marketing tool. It provides several useful features like sending transactional campaigns, email automation, and drip emails with customer journeys, WYSIWYG email editor, and many other tool integrations.
```

File: docs/destinations/streaming-destinations/mailmodo.mdx [hunk 7]
```diff
@@ -0,0 +1,158 @@
...
+
+<div class="infoBlock">
+Apart from Mailmodo destination, RudderStack also supports <a href ="/sources/event-streams/cloud-apps/mailmodo/">Mailmodo as a source</a>.
```
Comment (ameypv-rudder) [inline docs/destinations/streaming-destinations/mailmodo.mdx:10]:
```suggestion
RudderStack also supports Mailmodo as a data source. For more information, refer to the <a href ="/sources/event-streams/cloud-apps/mailmodo/">Mailmodo source documentation</a>.
```

File: docs/destinations/streaming-destinations/mailmodo.mdx [hunk 8]
```diff
@@ -0,0 +1,158 @@
+---
+title: "Mailmodo"
+description: Step-by-step guide on sending your event data from RudderStack to Mailmodo.
+aliases: ["Mailmodo destination"]
+---
+
+[Mailmodo](https://www.mailmodo.com/) is an interactive email marketing tool that provides several useful features such as sending transactional campaigns, email automation and drip emails with customer journeys, WYSIWYG email editor, and many other tool integrations.
+
+<div class="infoBlock">
```
Comment (ameypv-rudder) [inline docs/destinations/streaming-destinations/mailmodo.mdx:9]:
```suggestion
<div class="successBlock">
```

File: docs/destinations/streaming-destinations/mailmodo.mdx [hunk 9]
```diff
@@ -0,0 +1,158 @@
...
+
+<img src="../../assets/event-stream-destinations/mailmodo-connection-settings.png" alt="Mailmodo connection settings" />
+
+- **API Key**: Enter your Mailmodo API key.
+
+<div class="infoBlock">
+For more information on obtaining your Mailmodo API key, refer to the <Link to="#FAQ">FAQ</Link> section below.
+</div>
+
+- **List Name**: This field is **only applicable** for the `identify` call. Enter the name of the Mailmodo list where contacts should be added. If not provided, `Rudderstack` is taken as the default list name.
```
Comment (ameypv-rudder) [inline docs/destinations/streaming-destinations/mailmodo.mdx:43]:
```suggestion
- **List Name**: This field is **only applicable** for the `identify` call. Enter the name of the Mailmodo list where the contacts should be added. If not provided, `Rudderstack` is taken as the default list name. 
```

**Pull request merged**